apps/server/src/modules/provisioning/strategy/oidc/service/schulconnex-group-provisioning.service.spec.ts

```diff
@@ -760,6 +774,64 @@ describe(SchulconnexGroupProvisioningService.name, () => {
 					expect(result).toHaveLength(0);
 				});
 			});
+			describe('when group is empty after removal of the User but synced with a course', () => {
```

Comment:
Please remember the test guidelines https://documentation.dbildungscloud.dev/docs/schulcloud-server/Coding-Guidelines/testing.

Add empty lines between describe-blocks and before each return statement.